plugin.video.s04tv/default.py

```diff
@@ -34,418 +43,160 @@
 sys.path.append( os.path.join( BASE_RESOURCE_PATH, "lib" ) )
 sys.path.append( os.path.join( BASE_RESOURCE_PATH, "lib", "BeautifulSoup" ) )
 
-from BeautifulSoup import BeautifulSoup
-from BeautifulSoup import Tag
-import mechanize
 
 __language__ = __addon__.getLocalizedString
```

Comment:
Using names with __double_underscores__ is not recommended because they are reserved for internal Python names.

Reply:
Ok, thanks. I already changed some issues that codacy reported and will fix the things that you named as well.